g2p/convert.py

```diff
@@ -16,14 +16,16 @@ class BlissLexiconToG2PLexiconJob(Job):
     Convert a bliss lexicon into a g2p compatible lexicon for training
     """
 
-    def __init__(self, bliss_lexicon):
+    def __init__(self, bliss_lexicon, all_phons=False):
         """
         :param Path bliss_lexicon:
         """
         self.bliss_lexicon = bliss_lexicon
 
         self.out_g2p_lexicon = self.output_path("g2p.lexicon")
 
+        self.all_phons = all_phons
```

Comment:
this needs to be put above the output definition, see https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTUNG.md